server/Dockerfile

```diff
@@ -19,26 +20,31 @@ FROM "${DOCKER_HUB_PROXY}debian:bullseye-slim" as composer-build
         php-gd \
         php-fpm \
         php-zip \
+        unzip \
         && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
     
     WORKDIR /tmp
     ADD https://raw.githubusercontent.com/MISP/MISP/${MISP_COMMIT:-${MISP_TAG}}/app/composer.json /tmp
     COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
     RUN composer config --no-interaction allow-plugins.composer/installers true
     RUN composer install
-    RUN composer require --with-all-dependencies \
+    RUN composer require --with-all-dependencies --no-interaction \
             supervisorphp/supervisor:^4.0 \
             guzzlehttp/guzzle \
             lstrojny/fxmlrpc \
             php-http/message \
-            php-http/message-factory
-    RUN composer require --with-all-dependencies \
+            php-http/message-factory \
+            # docker image specific dependencies
             elasticsearch/elasticsearch:^8.7.0 \
             jakub-onderka/openid-connect-php:^1.0.0 \
             aws/aws-sdk-php
 
 FROM "${DOCKER_HUB_PROXY}debian:bullseye-slim" as php-build
     ENV DEBIAN_FRONTEND noninteractive
+    ENV TZ Etc/UTC
+    RUN apt-get update && apt-get install -y --no-install-recommends gpg-agent software-properties-common && add-apt-repository -y ppa:ondrej/php \
```

Comment:
Because of compliance we might want not to rely on PPAs unfortunately. Any alternative to this?

Reply:
Yes indeed, I missed this. Pushed a commit to remove.